src/pages/Graph/SummaryPanelCommon.tsx

```diff
@@ -173,7 +173,7 @@ export const renderNoTraffic = (protocol?: string) => {
   return (
     <>
       <div>
-        <Icon type="pf" name="info" /> No {protocol ? protocol : ''} traffic logged.
+        <KialiIcon.Info /> No {protocol ? protocol : ''} traffic logged.
```

Comment:
hmm.. this one is rendering blue and somewhat small:
![image](https://user-images.githubusercontent.com/23639005/67593172-2d75f080-f727-11e9-827b-2b539557acef.png)

Reply:
The default size is 10px I think.  Also, maybe it is wrong but PF4 info color is blue.  If we don't want blue we should change the definition in KialiIcon.

Reply:
@jshaughn shouldn't we be using whatever the PF4 Info color says?

Reply:
We are, we're using PfColors.Info for the info icon.  I think that is probably correct.  Edgar was wondering why it's blue because it used to be the default gray.

Reply:
Ok. Blue it is!